pandas/tests/frame/test_analytics.py

```diff
@@ -1892,6 +1892,25 @@ def test_clip_against_series(self, inplace):
 
             tm.assert_series_equal(clipped_df.loc[mask, i], df.loc[mask, i])
 
+    @pytest.mark.parametrize("inplace", [True, False])
+    def test_clip_against_list(self, inplace):
+        # GH #15390
+        original = self.simple
```

Comment:
Hmm, `self.simple` is supposed to be read-only. Can you just explicitly copy it here, since your modifying it later with `inplace=True`? `original = self.simple.copy()`